lms/djangoapps/labster_vouchers/views.py

```diff
@@ -91,9 +135,11 @@ def activate_voucher(request):
         return redirect(enter_voucher_url)
 
     code = form.cleaned_data['code']
+    anon_uid = anonymous_id_for_user(request.user, course_id)
+    context_id = course_id.to_deprecated_string()
```

Comment:
Just wonder does `to_deprecated_string` mean slash-separated string?

Reply:
str(course_id) should also work.

Reply:
Just moved the code.